pkg/drivers/hyperv/hyperv_windows.go

```diff
@@ -373,3 +376,34 @@ func (d *Driver) GetIP() (string, error) {
 
 	return resp[0], nil
 }
+
+func (d *Driver) GetSharedDirs() ([]drivers.SharedDir, error) {
+	dirs := []drivers.SharedDir{}
+	for _, dir := range d.SharedDirs {
+		if !smbShareExists(dir.Tag) {
+			return []drivers.SharedDir{}, drivers.ErrNotSupported
+		}
+
+		dir.Target = convertToUnixPath(dir.Target)
+		dir.Type = "cifs"
+		dir.Tag = smbShareUNCPathFromTag(dir.Tag)
+
+		dirs = append(dirs, dir)
+	}
+	return dirs, nil
+}
+
+func convertToUnixPath(path string) string {
+	// converts a path like c:\users\crc to /mnt/c/users/crc
+	return fmt.Sprintf("/mnt/c%s", strings.TrimLeftFunc(filepath.ToSlash(path), func(r rune) bool {
+		return (r == ':' || unicode.IsLetter(r))
+	}))
+}
+
+func smbShareUNCPathFromTag(tag string) string {
+	hostname, err := cmdOut("hostname")
+	if err == nil {
+		return fmt.Sprintf("//%s.mshome.net/%s", strings.TrimSpace(hostname), tag)
```

Comment:
Wondering if we could use an IP there instead of hostname.mshome.net? Not sure it's better, maybe it would make the code simpler?

Reply:
we have to figure out the ip address on one of the interfaces of the host or the "Default Switch" the default switch name might be localized, but this gives the ip address for default switch `(Get-NetIPAddress -InterfaceAlias "vEthernet (Default Switch)").IPv4Address`

Reply:
In minishift we extracted the IP address, verified reachability before allowing it to be used.

Reply:
changed to use an ip address instead of hostname.mshome.net for the UNC path